hw04/BarbieWalletBalance.cs

```diff
@@ -13,24 +13,51 @@ public class BarbieBank : MonoBehaviour
     BarbieBank should take in the parameters: numOfPennies, cashAmount, and numOfCreditCards
     Have the function return a new generic array with the correct parameters. 
     */
-
     void Start()
     {
+        
```

Comment:
We will need to inherit from `BarbieWorld.cs`. We write the name of the class and then the name of the method followed by the brackets. 

```
GenericClass<int> myClass = new GenericClass<int>();
```

or for this problem:

```
BarbieWorld<string> barbieWorld = new BarbieWorld<string>();
barbieWorld.BarbieCareers("FashionModel");
```